commit 828e10cb9331fc5bd48fb1d6fdb8e08cd5fb27ea
Author:     Thomas Deutschmann <whissi@gentoo.org>
AuthorDate: Mon Apr 29 18:57:10 2019 +0200
Commit:     Thomas Deutschmann <whissi@gentoo.org>
CommitDate: Mon Jun 24 23:52:25 2019 +0200

    configure: Add option to disable automagic dependency on zstd
    
    This commit will add an option which will allow you to explicit disable
    zstd usage.

diff --git a/mklove/modules/configure.libzstd b/mklove/modules/configure.libzstd
index d9599d82..cb82c128 100644
--- a/mklove/modules/configure.libzstd
+++ b/mklove/modules/configure.libzstd
@@ -9,8 +9,12 @@
 #   mkl_check libzstd [<action>]
 #
 
+mkl_toggle_option "Feature" ENABLE_ZSTD "--enable-zstd" "Enable support for ZSTD compression" "y"
+
 function manual_checks {
-    local action=$1
+    local action=${1:-disable}
+
+    [[ $ENABLE_ZSTD == y ]] || return 0
 
     mkl_meta_set "libzstd" "brew" "zstd"
     mkl_meta_set "libzstd" "apk" "zstd-dev zstd-static"
